examples/make_import/README.md

```diff
@@ -24,8 +24,6 @@ Use the common help target:
         ENV       | VALUE |   DEFAULT   |                            SHORT DESCRIPTION                              
 ------------------+-------+-------------+---------------------------------------------------------------------------
   ARCH            |       | amd64       | Set the arch (Default from make)                                          
-  DEBIAN_FRONTEND |       |             | Set to noninteractive or stderr to null to enable non-interactive output  
```

Comment:
did DEBIAN_FRONTEND get removed? I originally put this in to emulate the behaviour that many debian tools have that will pick up on this env var and change the output mode. I'm not aware of a generic env so this one seemed the most approrpiate.

Reply:
Yes, I removed it but like I've been saying above I could reintroduce this and derive a default for it if I reverted the changes and moved environment back to package globals.
I know it's about configuration by convention more than anything - but I thought that having an [explicit configuration](https://github.com/gravitational/magnet/pull/2/files#diff-20346fa97ecbc96a69f300130936b45bdc4bfc57a73a204792178fb1d680067eR41) option would make the need for the implicit configuration less of an issue

Reply:
Yea, the reason I hacked it up this way was to have configuration by convention, if you walk into any of these projects for some of the main interactions they work the same way. There are probably ways to approach it like move the vars into root and having a default that can optionally be called by the config when loaded or something. 

But I think having a consistent way to turn the interactive mode off will come up if/when this get's wired to a CI/CD type pipeline, to get usable logs on stdout.

Reply:
I'm not opposed to making these MAGNET_ vars if that's the preferred idiom than trying to find something that makes sense in unix/mac lands, so `MAGNET_CACHE=...` / `MAGNET_OUTPUT_MODE=<noninteractive | interactive>`. Using a mode might make sense if we want to add more output formats in the future, like say json to feed to centralized logging.

Reply:
Reverted the `DEBIAN_FRONTEND` and `XDG_CACHE_HOME` to keep the configuration-by-convention spirit and used them to infer the value for the progress flag. True, it is currently boolean but I'd keep it that way since this is as much as is needed from it (although buildkit does have it as a mode).